Source/System/Drawing/ColorConverter.cs

```diff
@@ -0,0 +1,433 @@
+                            }
+                            tryMappingToKnownColor = true;
+                        }
+
+                        if ((obj != null) && tryMappingToKnownColor) {
+
+                            // Now check to see if this color matches one of our known colors.
+                            // If it does, then substitute it.  We can only do this for "Colors"
+                            // because system colors morph with user settings.
+                            //
+                            int targetARGB = ((Color)obj).ToArgb();
+
+                            foreach (Color c in Colors.Values) {
+                                if (c.ToArgb() == targetARGB) {
+                                    obj = c;
+                                    break;
+                                }
+                            }
+                        }
+                    }
+#if NETSTANDARD20
+                    if (obj == null)
+                    {
+                        throw new ArgumentException(nameof(value));
+                    }
+#else
+                    if (obj == null) {
+                        throw new ArgumentException(SR.Format(SR.InvalidColor, text));
+                    }
+#endif
+                }
+                return obj;
+            }
+            return base.ConvertFrom(context, culture, value);
+        }
+
+        /// <include file='doc\ColorConverter.uex' path='docs/doc[@for="ColorConverter.ConvertTo"]/*' />
+        /// <devdoc>
+        ///      Converts the given object to another type.  The most common types to convert
+        ///      are to and from a string object.  The default implementation will make a call
+        ///      to ToString on the object if the object is valid and if the destination
+        ///      type is string.  If this cannot convert to the desitnation type, this will
+        ///      throw a NotSupportedException.
+        /// </devdoc>
+        public override object ConvertTo(ITypeDescriptorContext context, CultureInfo culture, object value, Type destinationType) {
+            if (destinationType == null) {
+                throw new ArgumentNullException(nameof(destinationType));
+            }
+
+            if ( value is Color ){
+                if (destinationType == typeof(string)) {
+                    Color c = (Color)value;
+
+                    if (c == Color.Empty) {
+                        return string.Empty;
+                    }
+                    else {
+                        // If this is a known color, then Color can provide its own
+                        // name.  Otherwise, we fabricate an ARGB value for it.
+                        //
+#if NETSTANDARD20
+                        if (c.IsNamedColor) {
+                            return "'" + c.Name + "'";
+                        }
+#else
+                        if (c.IsKnownColor) {
+                            return c.Name;
+                        }
```

Comment:
So basically you have removed this check - don't we have `IsKnownColor` (and `IsSystemColor` below)?

Reply:
It's also removed using if/def when using netstandard2.0 target framework.

Reply:
Ah ok, obviously it's not defined there. Anyway, this is not important, as it just means that hex codes will be used instead of color names.